packages/app/src/vis-packs/nexus/containers/NxRgbContainer.tsx

```diff
@@ -16,26 +19,45 @@ function NxRgbContainer(props: VisContainerProps) {
   assertNumericSignal(nxData);
 
   const { signalDef } = nxData;
-  assertNumDims(signalDef.dataset, 3);
+  assertMinDims(signalDef.dataset, 3);
 
   const { shape: dims } = signalDef.dataset;
+  if (dims.at(-1) !== 3) {
```

Comment:
Nice, I just discovered [at](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Array/at) :joy_cat:

Reply:
Yeah, thought I'd take it for a spin. We'll have to remember to check if we need to add a polyfill in Braggy/Hibou/....